styles/styles.css

```diff
@@ -1,21 +1,327 @@
 body {
   margin: 0;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
 }
 
 .toolbar {
   display: flex;
   justify-content: space-between;
-  padding: 0;
 }
 
-.navbar-image {
-  margin-right: 15px;
-  margin-top: 70%;
-  width: 20px;
-  height: 14px;
+.toolbar .navbar-icon {
+  margin-right: 16px;
+}
+
+@media (min-width: 992px) {
```

Comment:
- [ ] You did a good job applying the media queries needed for this project, but let's try to avoid repetition as much as possible. One of the most important "rules" to follow while coding it's [DRY](https://qvault.io/clean-code/dry-code/), so please make sure to avoid the repetition of `@media (min-width: 992px)` all over your CSS file. Using just one it's more than enough because you can wrap everything inside like `@media (min-width: 992px) { put all the code here }` and that will run perfectly.